Designer/ViewModel/ViewDesigns.cs

```diff
@@ -59,12 +59,12 @@ public class DesignCatalogModel : INotifyPropertyChanged {
             set => GotoDesign(value);
         }
 
-        public DesignCatalogModel() {
+        public ViewDesigns() {
             Navigator = Navigator.Instance;
             Rooms = LoadRooms();
             EditCommand = new ArgumentCommand<Design>(design =>
             {
-                Navigator.Push(new ViewDesignPage((Design)design.Clone()));;
+                Navigator.Push(new ViewDesignPage(design));;
```

Comment:
Is het wat om de pagina namen ook te veranderen naar DesignEditor(Page?)

Reply:
het de view classes aangepast naar [ViewNaam]View, anders hebben de view en de viewmodel dezelfde classnaam wat weer kut is